dags/utils/helpers.py

```diff
@@ -19,22 +19,26 @@ def get_postgres_uri(name):
     )
 
 
-def create_collector_task(name, dag, environment=None):
+def create_collector_task(name, dag, command=None, environment=None):
+    default_command = 'make start {}'.format(name)
+
     return _create_task(
         task_id='collector_{}'.format(name),
         dag=dag,
         image='okibot/collectors:latest',
-        command='make start {}'.format(name),
+        command=command or default_command,
```

Comment:
It'll probably be useful to be able to change the commands without redeploying the container. However, it also makes things a bit more difficult to track, as we would have to check not only the DAG code, but also what variable it's set as.

Considering the DAGs shouldn't be changing often after we pass through this initial phase of setting things up, I think we'll be fine hardcoding the commands. However, it's good to keep this idea in mind in case we find ourselves changing the code a lot just to tweak the commands.